biasplt.py

```diff
@@ -0,0 +1,88 @@
+import sys
```

Comment:
I suggest sticking with the naming conventions. Langmuir Plot -> lplt, Bias plot -> bplt. 
Its not a perfect convention, but it should be consistent.